config/v1/0000_10_config-operator_01_infrastructure-CustomNoUpgrade.crd.yaml

```diff
@@ -466,6 +466,13 @@ spec:
                     aws:
                       description: AWS contains settings specific to the Amazon Web Services infrastructure provider.
                       properties:
+                        clusterHostedDNS:
+                          default: Disabled
+                          description: ClusterHostedDNS indicates if the customer is providing their own DNS solution in place of the default provided by Core DNS.
```

Comment:
Description has to change to:
ClusterHostedDNS indicates the type of DNS being used by the cluster. When `Enabled` the cluster runs a DNS service within the cluster. The default value of "Disabled" indicates that the cluster is utilizing the DNS solution provided by the underlying cloud. This value is set by the Installer and cannot be changed during the life of the cluster.